components/browser/toolbar/src/main/res/values-de/strings.xml

```diff
@@ -6,9 +6,9 @@
     <!-- Content description: For the tracking protection toolbar icon, it is set when the site has tracking protection enabled, but none trackers have been blocked or detected. -->
     <string name="mozac_browser_toolbar_content_description_tracking_protection_on_no_trackers_blocked">Schutz vor Aktivitätenverfolgung ist an</string>
     <!-- Content description: For the tracking protection toolbar icon, it is set when the site has tracking protection enabled, and trackers have been blocked or detected.-->
-    <string name="mozac_browser_toolbar_content_description_tracking_protection_on_trackers_blocked">Schutz vor Aktivitätenverfolgung ist aktiv und blockiert Tracker</string>
+    <string name="mozac_browser_toolbar_content_description_tracking_protection_on_trackers_blocked1">Der Tracking-Schutz hat Tracker blockiert</string>
```

Comment:
With this change the wording no longer consistent with other Firefox products…

Reply:
@Pike @Delphine What's your recommendation here?

Reply:
CC @MichaelKohler

Reply:
Localization reviews shouldn't happen in these PRs, these are really just for exporting the data from one repo to the other.

The goal for these PRs is to automatically merge once tests pass, which is actively worked on, so there's no point in establishing other process in the meantime.

Reply:
Merging 👍

Reply:
> Localization reviews shouldn't happen in these PRs

@Pike What is the alternative? It's not possible to discuss translations on Pontoon at all. And since it's not the first time that there is a need to discuss a translation it needs a place to have this kind of conversation.